test/cpp/jit/test_lite_interpreter.cpp

```diff
@@ -756,6 +756,36 @@ TEST(LiteInterpreterTest, GetRuntimeOpsAndInfo) {
   AT_ASSERT(runtime_ops.size() > 2900);
 }
 
+TEST(LiteInterpreterTest, isCompatibleSuccess) {
+  // test trivial success case
+  auto runtime_info = get_runtime_compatibility_info();
+  std::unordered_map<std::string, OperatorInfo> model_ops;
+  model_ops["aten::add.Scalar"] = OperatorInfo{2};
+  // arbitrarily pick bytecode version 5. This may need to be updated some time
+  // in the future
+  auto model_info = ModelCompatibilityInfo{0x5, model_ops};
```

Comment:
Consider use `caffe2::serialize::kProducedBytecodeVersion`. Then it's expected to be always compatible.